### ISSUE TITLE
How to set up a fee for funding tx

### ISSUE THREAD

**Issue by yuyaogawa** [LightningPeach/lightning-peach-wallet#11]
There is no way to set up a fee to open a channel. When a funding transaction is created and broadcasted with small fee, it got stack and never get confirmed.

**Comment by boodyvo** [comment]
Yes, we know it is a problem and we are working on functionality to set fee level for onchain transactions and for channel open.
